src/server/scripts/Outland/BlackTemple/boss_illidan.cpp

```diff
@@ -979,6 +989,7 @@ class boss_illidan_stormrage : public CreatureScript
                         break;
                     case EVENT_DEFEATED_TEXT:
                         Talk(SAY_ILLIDAN_DEFEATED);
+                        me->GetMap()->SetZoneMusic(BLACK_TEMPLE_ZONE_ID, EVENT_BT_ARRIVAL_WALK_HERO_1_SOUND_ID);
```

Comment:
Perhaps (at least) this sound should be in the creature_text soundId field for `SAY_ILLIDAN_DEFEATED`?